### ISSUE TITLE
Gang File Problems when Both Necromunda Underhive and Necromunda Gang Wars files are downloaded 

### ISSUE THREAD

**Issue by luminositeboy** [BSData/necromunda-gang-war#14]

**Comment by cjo197** [comment]
It looks like two repositories have been created; a Necromunda Underhive one for the basic boxed game and this Necromunda Gang War one for all the campaign stuff in the Gang War books. But they have the same unique id so they clash with each other. For now I suggest you go to Manage Data, make sure you only have Necromunda Gang War selected, then hit refresh and tick 'Delete existing data first'.

@amis92 I'm still quite new to this. What's the best way to fix it properly? Can I just change the id in the .gst in this repo or will that cause problems for everyone that's already downloaded it?

**Comment by amis92** [comment]
I'm not sure what should be the best approach. Is there even a need for these two to coexist?

Changing the gst ID will cause problems, indeed. It's the last resort. Could we maybe delete/archive the underhive repo and redirect people to this one?

**Comment by cjo197** [comment]
I'd be happy with that solution. I can't imagine many people would still be creating custom gangs from the starter box for one off games.

**Comment by amis92** [comment]
Can you summarize the differences between the two for me? Is this a strict superset of underhive, or are there changes, or worse, removal of things too?

**Comment by cjo197** [comment]
In terms of files, yes. Underhive has the gst and a cat each for Escher and Goliath. Gang War has those three with the same root ids plus more cats.

I think the initial load for Gang War already had a bunch of changes over Underhive but was probably at least started from a copy of the latest Underhive files. There have been more changes in those three files in Gang War since then. It's mostly been text/point changes or adding new stuff.

Underhive hasn't been updated since long before Gang War was started so we won't be losing any changes.

**Comment by OftKilted** [comment]
Same issue as #27

**Comment by OftKilted** [comment]
Symptom of the issue is that Esher and Goliath juves are not available Escher juve in Gang War not available
